augerctl/README.md

```diff
@@ -62,10 +62,10 @@ TODO
 List a single service with namespace `default` and name `kubernetes`
 
 ``` bash
-augerctl get services -n default kubernetes
+augerctl get svc -n default kubernetes
```

Comment:
Why would the short name replace the long one? Shouldn't both be supported?

Reply:
Both uses are supported, but I think the descriptions of both uses are a bit verbose.